MetaMask/metamask-extension
# feat: Add ConnectedAccountsMenu Component

## **Description**

Adds ConnectedAccountsMenu Component

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23228?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/23197

## **Manual testing steps**

- Add the component somewhere in the DOM
- Check if it renders, as it has no functionality yet

## **Screenshots/Recordings**

<img width="480" alt="Screenshot 2024-02-28 at 01 37 36" src="https://github.com/MetaMask/metamask-extension/assets/11148144/b1d22120-c18e-4657-a2f4-37b88a709949">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
